inc/field.php

```diff
@@ -105,10 +105,12 @@ public static function begin_html( $meta, $field ) {
 			);
 		}
 
+		$data_min_clone = is_numeric( $field['min_clone'] ) && $field['min_clone'] > 1 ? ' data-min-clone=' . $field['min_clone'] : '';
```

Comment:
I think the check here should be `$field['min_clone'] > 0`

Reply:
The default 'min_clone' value is 1. 
if no value is passed, no value is set, and the default is 1
if a value of 1 is passed, no value is set, as that is the default value
if a value greater than 1 is passed, then the value is passed and the default changed.
It would work either way, but why pass a value if it will have no effect?

Reply:
You're right! It's better to eliminate the `data-*` when it's the default value. Good point!